mpas_analysis/sea_ice/timeseries.py

```diff
@@ -310,45 +312,79 @@ def seaice_timeseries(config, streamMap=None, variableMap=None):
             if (compareWithObservations or
                     preprocessedReferenceRunName != 'None'):
                 # separate plots for nothern and southern hemispheres
-                timeseries_analysis_plot(config, varsNH, movingAveragePoints,
-                                         titleNH,
-                                         xLabel, units, figureNameNH,
-                                         lineStyles=lineStyles,
-                                         lineWidths=lineWidths,
-                                         titleFontSize=titleFontSize,
-                                         calendar=calendar)
-                timeseries_analysis_plot(config, varsSH, movingAveragePoints,
-                                         titleSH,
-                                         xLabel, units, figureNameSH,
-                                         lineStyles=lineStyles,
-                                         lineWidths=lineWidths,
-                                         titleFontSize=titleFontSize,
-                                         calendar=calendar)
+                if (polarPlot):
+                    timeseries_analysis_plot_polar(config, varsNH, movingAveragePoints,
+                                                   titleNH,
+                                                   figureNameNH,
+                                                   lineStyles=lineStyles,
+                                                   lineWidths=lineWidths,
+                                                   titleFontSize=titleFontSize,
+                                                   calendar=calendar)
+                    timeseries_analysis_plot_polar(config, varsSH, movingAveragePoints,
+                                                   titleSH,
+                                                   figureNameSH,
+                                                   lineStyles=lineStyles,
+                                                   lineWidths=lineWidths,
+                                                   titleFontSize=titleFontSize,
+                                                   calendar=calendar)
+                else:
```

Comment:
I think that's weird because then you accidentally turn both off and do a bunch of computing that doesn't produce a plot and you wonder what went wrong.  I realized that's probably a really unlikely case.  But this is something I had to think through with the different modes of ice-ocean interactions in MPAS-Ocean (whether land-ice pressure depresses the surface, whether melt fluxes are computed and whether those fluxes come from the coupler or are computed in the ocean model).  Boolean flags could leave you in a meaningless state and it's not always obvious to the user which combinations are allowed and which aren't.

TL, DR: it's complicated...